pyro/distributions/transforms/batchnorm.py

```diff
@@ -127,3 +127,19 @@ def log_abs_det_jacobian(self, x, y):
             # NOTE: You wouldn't typically run this function in eval mode, but included for gradient tests
             var = self.moving_variance
         return (-self.constrained_gamma.log() + 0.5 * torch.log(var + self.epsilon))
+
+
+def batchnorm(input_dim, **kwargs):
```

Comment:
do we really want trivial helpers like this one?

Reply:
I think yes! The point of the helpers is that they give you a default which doesn't require you to know any of the details of how the transformation is parametrized. If you had to know that the batch-norm doesn't take a hypernet then that would defeat the point. Also, if there were conditional versions of batch-norm, householder, etc. they would take a hypernet and have a non-trivial helper function

Reply:
I think it's fine to add trivial helpers, but only if you add some documentation section somewhere with a menu of all available helpers with links to their docs.

Reply:
I've added a sidebar for helpers in docs